src/presentation/controller/addressLookUp/Controller.ts

```diff
@@ -83,7 +92,7 @@ class AddressLookUpController extends AbstractController {
   ): Promise<Address[]> {
     let addressList: Address[] = [];
 
-    if (this.isAddressListRequired(pageRouting.pageType)) {
+    if (AddressLookUpController.ADDRESS_LIST_REQUIRED_PAGES.has(pageRouting.pageType)) {
```

Comment:
Do you need the '`AddressLookUpController.`' prefix? You wouldn't in Java.

Reply:
yes, statics in typescript need it unless calling from static code